articles/portal/ptl-how-reset-2fa.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to reset two-factor authentication for a user account
+description: Describes how to reset two-factor authentication (2FA) for a user account
+services: portal
+author: Sue Highmoor
+reviewer: 
+lastreviewed: 10/02/2020
+
+toc_rootlink: How To
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Reset 2FA for a user account
+toc_fullpath: How To/ptl-how-reset-2fa.md
+toc_mdlink: ptl-how-reset-2fa.md
+---
+
+# How to reset two-factor authentication for a user account
+
+## Overview
+
+If two-factor authentication (2FA) is enabled for the UKCloud Portal, users must use an authenticator app to generate a code to enter, as well as their user name and password, when logging in to the Portal.
+
+If a user loses or replaces the device where their authenticator app is installed, they will be unable to generate the required code and therefore be unable to log in to the Portal. In this situation, an administrator must reset the user's 2FA so that they can set up 2FA on a different device.
+
+### Intended audience
+
+This article is intended for Portal administrators who need to reset a Portal user's 2FA.
+
+## Resetting 2FA for a user account
+
+1. Log in to the UKCloud Portal as a Portal administrator.
```

Comment:
Is it worth adding that the Admin must also have 2FA enabled for them to see the reset button?

Reply:
Done